features/build_runner/build_runner.rb

```diff
@@ -62,7 +62,7 @@ def setup
     def start
       start_time = Time.now
 
-      logger.debug("Running runner now")
+      logger.debug("Starting runner #{self.class} now...")
```

Comment:
instead of self.class (already reported as part of logger.debug) let's use `self.project.name` or something like that so we can see what is happening easier when we have a bunch of runners

Reply:
Ah yeah